site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -72,7 +72,7 @@ small resource requesting workloads admitted at the same time.
 
 It is possible that multiple resources in a ClusterQueue have the same flavors.
 This is typical for `cpu` and `memory`, where the flavors are generally tied to
-a machine family or VM availability policies. To tie two or more resources to 
+a machine family or VM availability policies. To tie two or more resources to
```

Comment:
I think these spaces are intended.

Reply:
Why? we have a newline here.

Reply:
Even If we have a new line, we used to add space like in most places. For example, you can see in Line 69.
If we want to remove these spaces, we need to work on all the places in the documentation.

Reply:
> we used to add space like in most places

Do we? What I saw is most of the lines doesn't have the space.

Reply:
I checked spaces in https://raw.githubusercontent.com/kubernetes-sigs/kueue/91ee062d4e5f63dfb5b7b5aec5139c45a5156555/site/content/en/docs/concepts/cluster_queue.md.

And then, most places have a space. However, once I confirmed spaces in my local, as you said, there aren't spaces in most places.

It looks like the GitHub adds space to all places...